src/services/itemMembership/service.ts

```diff
@@ -179,7 +218,8 @@ export class ItemMembershipService {
     }
     const { itemMembershipRepository } = repositories;
     // check memberships
-    const { item } = await itemMembershipRepository.get(itemMembershipId);
+    const membership = await itemMembershipRepository.get(itemMembershipId);
+    const item = membership.item;
```

Comment:
```suggestion
    const { item } = membership;
```

Reply:
I think destructuring is often confusing as a syntax and only improves readability in specific cases, but rarely for one element :D I'll follow the customs though